.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 120 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
How does this work - if the issue is marked as stale, and I remove the `stale` label, will it be marked stale again in 120 days?

Reply:
Yes

Reply:
I'm just thinking we have issues that we should do but in the major version or something like that. So we'll be re-triaging every 3 months.